src/promptflow-devkit/promptflow/_sdk/_orchestrator/run_submitter.py

```diff
@@ -212,8 +213,19 @@ def _submit_bulk_run(
             local_storage.persist_result(batch_result)
             # exceptions
             local_storage.dump_exception(exception=exception, batch_result=batch_result)
-            # system metrics: token related
-            system_metrics = batch_result.system_metrics.to_dict() if batch_result else {}
+            # system metrics
+            system_metrics = {}
+            if batch_result:
+                system_metrics.update(batch_result.system_metrics.to_dict())  # token related
+                system_metrics.update(
+                    {f"{SystemMetricKeys.NODE_PREFIX}.{k}": v for k, v in batch_result.node_status.items()}
```

Comment:
Will flex flow have this metric?

Reply:
Yes, as long as its metrics are stored in the `batch_result`, if not it should be a bug of batch engine.